controllers/akodeploymentconfig/akodeploymentconfig_controller_avi_phase.go

```diff
@@ -6,15 +6,11 @@ package akodeploymentconfig
 import (
 	"bytes"
 	"context"
-	"net"
-	"sort"
-	"time"
-
-	"github.com/pkg/errors"
-
 	"github.com/vmware-samples/load-balancer-operator-for-kubernetes/controllers/akodeploymentconfig/phases"
 	"github.com/vmware-samples/load-balancer-operator-for-kubernetes/controllers/akodeploymentconfig/user"
 	"github.com/vmware-samples/load-balancer-operator-for-kubernetes/pkg/haprovider"
+	"net"
```

Comment:
Did you install gofmt or other golang plugin, it should format the import for you as well

Reply:
consider adding rules in `golangci-lint` ?

Reply:
fix the import order for now. I am using Goland but somehow the import order was wrong.